quickstarts/microsoft.dataprotection/backup-create-storage-account-enable-protection/main.bicep

```diff
@@ -43,28 +80,179 @@ resource backupVault 'Microsoft.DataProtection/backupVaults@2021-01-01' = {
   }
 }
 
-resource backupPolicy 'Microsoft.DataProtection/backupVaults/backupPolicies@2021-01-01' = {
-  parent: backupVault
-  name: backupPolicyName
+resource vaultName_backupPolicy 'Microsoft.DataProtection/backupVaults/backupPolicies@2022-05-01' = {
```

Comment:
this name looks like it came from the decompiler and is not particularly friendly. Should this change back to simply `backupPolicy`?

Reply:
Addressed